runtime/syntax/bend.yaml

```diff
@@ -0,0 +1,49 @@
+# Author: https://github.com/RohanVashisht1234
+filetype: bend
+
+detect:
+    filename: "\\.bend$"
+
+rules:
+      # keywords
+    - statement: "\\b(def|switch|case|return|if|else|when|match|λ|Some|data|let|use|object|fold|open|do|bind|Name|identity|Bool|ask|with)\\b"
+      # special method names
+    - symbol.operator: "\\b(bend|None|Nil|Result|type|lambda)\\b"
+      # definitions
+    - identifier: "def [a-zA-Z_0-9]+"
+      # keywords (declaration)
+    - statement: "\\b(Node|Leaf|Tree)\\b"
+      # constants (true, false, null, this)
+    - constant: "\\b(true|false|True|False)\\b"
+      # constants (types)
+    - type: "\\b(ud|str|int|float|obj|fib|bool|pat|type|any)\\b\\??"
```

Comment:
I tried looking at this file but I do not think these types are in Bend:
https://github.com/HigherOrderCO/Bend/blob/main/docs/builtins.md

Reply:
Sorry added -type thing, but can we keep true false? because people do use true false in bend by declaring this on their side.

Reply:
Sorry, I was not trying to refer to `true` and `false` but I realized there is `null` and `this` in the comment even though those are not highlighted. The types at the link I referenced can still be higlighted.

Reply:
ok, fixed that because the `null` and `this` isn't a actual type in bend afaik